src/tests/pages/pages.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {
+  render, screen,
+} from '@testing-library/react';
+import '@testing-library/jest-dom';
+import CalculatorPage from '../../pages/CalculatorPage/index.js';
+import HomePage from '../../pages/HomePage/index.js';
+import QuotePage from '../../pages/QuotePage/index.js';
+
+describe('Pages', () => {
+  it('Should render HomePage', async () => {
+    render(<HomePage />);
+    expect(screen.getByText('Welcome to our page!')).toBeInTheDocument();
+  });
+
+  it('Should render CalculatorPage', async () => {
+    render(<CalculatorPage />);
+    expect(screen.getByText('Lets do some math!')).toBeInTheDocument();
+  });
+
+  it('Should render QuotePage', async () => {
+    render(<QuotePage />);
+    expect(screen.getByText('Math Quote..')).toBeInTheDocument();
+  });
+});
```

Comment:
- [ ] There should be the usage of Jest snapshots for testing all the components.

After usage of Jest snapshots, the snapshot folder will be created with the components that have been tested

Reply:
I like this one! Thanks for the suggestions I had a very hard time trying to figure it out what was this snapshot.

I used **'react-test-renderer'**

Thanks!